static/src/javascripts/projects/common/modules/experiments/ab.js

```diff
@@ -53,13 +75,40 @@ export const getEpicTestToRun = memoize(
                     test => !test.highPriority
                 );
 
-                return firstRunnableTest<EpicABTest>([
+                const result = firstRunnableTest<EpicABTest>([
                     hardCodedPriorityEpicTest,
                     ...highPriorityConfiguredTests,
                     ...highPriorityHardCodedTests,
                     ...lowPriorityConfiguredTests,
                     ...lowPriorityHardCodedTests,
                 ]);
+
+                if (result) {
```

Comment:
Is no result (i.e. no runnable test) also an outcome we want to compare against? I.e. if frontend decides there isn't a runnable test, then we want the contributions service to agree with this right? If that's the case then maybe we need to remove this condition?